### ISSUE TITLE
Unable to Reproduce the Finetuning Performance

### ISSUE THREAD

**Issue by andrewch1312** [VITA-Group/Adversarial-Contrastive-Learning#19]
I'm trying to reproduce the adversarial finetuning part using the provided checkpoint "ACL_DS.pt" as the pre-trained initial weights.

I followed the exact settings in the paper and ran the command:

`python train_trades.py name --checkpoint ACL_DS.pt --cvt_state_dict --bnNameCnt 1 --data data --decreasing_lr 15,20 --epochs 25 --lr 0.1 --batch-size 128`

However the results I get is far from the reported performance:

**_On the ata_best_model, test tacc is 76.25999999999999, test atacc is 48.779998779296875_**
and 
**_On the best_model, test tacc is 80.67, test atacc is 45.46999740600586_**

Is there anything I can do to better reproduce the performance? or am I doing something wrong?
Thanks!

Note: I have modified the code of "train_trades.py" to use held-out robust accuracy instead of using validation data, which is how the paper evaluated the model.

**Comment by geekJZY** [comment]
Which PyTorch version are you using? We found that there would be a performance drop when using other Pytorch versions. Please use Pytorch 1.5.0 or 1.5.1 for reproducing our results.

**Comment by andrewch1312** [comment]
> Which PyTorch version are you using? We found that there would be a performance drop when using other Pytorch versions. Please use Pytorch 1.5.0 or 1.5.1 for reproducing our results.

Thank you so much for the quick reply!
I was using Pytorch 1.9, and sorry for not reading the description thoroughly.
I will run the experiment again in 1.5 and check the results.

Thanks again!

**Comment by priyamdey** [comment]
hey @andrewch1312 , quick follow up - were you able to reproduce the results with torch 1.5?
